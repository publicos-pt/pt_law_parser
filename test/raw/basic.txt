Decreta que:
Título I
Princípios e disposições comuns
Artigo 1º
Objecto e âmbito
1 - foobar
a) blabla
2 - foobar
Título II
Instituições, unidades orgânicas e ciclos de estudos
Capítulo I
Forma e procedimento de criação de instituições
Artigo 2º
Instituições de ensino superior públicas
bla bla bla Decreto-Lei 2/2002
