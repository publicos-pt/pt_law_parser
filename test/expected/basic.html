<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"></head>
<p>Decreta que:</p>
<div class="title"><h3 class="title">Título I
<p>Princípios e disposições comuns</p></h3>
<div class="article" id="Artigo-1º"><h5 class="title">Artigo <a href="#Artigo-1º">1º</a>
<p>Objecto e âmbito</p></h5><ol><li class="number list-unstyled" id="Artigo-1º-Número-1">
<span><a href="#Artigo-1º-Número-1">1 -</a></span>
<span> foobar</span><ol><li class="line list-unstyled" id="Artigo-1º-Número-1-Alínea-a)">
<span><a href="#Artigo-1º-Número-1-Alínea-a)">a)</a></span>
<span> blabla</span></li></ol></li><li class="number list-unstyled" id="Artigo-1º-Número-2">
<span><a href="#Artigo-1º-Número-2">2 -</a></span>
<span> foobar</span></li></ol></div></div>
<div class="title"><h3 class="title">Título II
<p>Instituições, unidades orgânicas e ciclos de estudos</p></h3>
<div class="chapter"><h3 class="title">Capítulo I
<p>Forma e procedimento de criação de instituições</p></h3>
<div class="article" id="Artigo-2º"><h5 class="title">Artigo <a href="#Artigo-2º">2º</a>
<p>Instituições de ensino superior públicas</p></h5>
<p>bla bla bla Decreto-Lei <a>2/2002</a></p></div></div></div></html>